### DISCUSSION TITLE
FL 1.10.0. Modify translation of the title Plugin and Hotkey. Proposal

### DISCUSSION THREAD

**Discussion by dracp** [Flow-Launcher/Flow.Launcher#1453] [Ideas]
Having seen the context of the Plugin and Hotkey strings in the graphical interface, I understand that both strings should be written in the plural. "Plugins" and "Hotkeys". Proposal translated into Spanish is attached.

![221006 FL 1 10 0_Propuesta_Complementos_Atajos](https://user-images.githubusercontent.com/102802425/194274980-b8daf64f-7a4a-473d-8b13-48a859ba2415.png)

**Comment by taooceros** [comment]
Please propose all these change to crowdin, which will be pushed back as a pull request.

**Comment by dracp** [comment]
As you can see in the figure above, in Crowdin, Plugin and Hotkey have been translated into Spanish in plural ("Complementos" and "Atajos de teclado").
Attached is a screenshot of the position.

![221007 Crowdin_Plugin_Hotkey_en plural](https://user-images.githubusercontent.com/102802425/194625981-067014f4-fd1b-4b7f-be8c-e73ca3b8e92f.png)

**Reply by taooceros** [reply]
I believe the first one is already merged (could you check the latest preview). But for the later, it is already in the pr, and we plan to merge it around one week from now (we don't want to have too much translation commit since their message are more or less the same).

**Comment by dracp** [comment]
It is understood. I will download FL 1.10.0 again and check.

**Comment by dracp** [comment]
**Revised FL 1.10.0, installed on October 08**
See:
https://github.com/Flow-Launcher/Flow.Launcher/discussions/1442#discussioncomment-3828118
